scripts/validate

```diff
@@ -6,6 +6,26 @@ if [ -n "${SKIP_VALIDATE}" ]; then
     exit 0
 fi
 
+function check_win_binaries() {
+    # --- disabled until upstream releases a 1.23 release of cri-tools
+    #CRICTL_WINDOWS_VERSION=$(grep 'CRICTL_VERSION=' Dockerfile.windows | cut -d '=' -f 2- | grep -oE "v([0-9]+)\.([0-9]+)")
+    #if [ ! "$CRICTL_WINDOWS_VERSION" = "v$MAJOR.$MINOR" ]; then
+    #    fatal "crictl windows binary version [$CRICTL_WINDOWS_VERSION] does not match kubernetes version"
+    #fi
```

Comment:
there are now 1.23 and 1.24 versions of cri-tools

Reply:
I'd rather remove those commented out lines, since I don't know if we want to bump `cri-tools` version in `Dockerfile.windows`

Reply:
I would recommend keeping them, and fixing the version in the Dockerfile. Now, between releases, is a good time to do that.